plugin/config.go

```diff
@@ -54,7 +55,9 @@ func (cm *defaultConfigManager) SetupConfig(cfg config.Configuration) error {
 		maxPubSub = config.DefaultConsulPubMaxRoutines
 	}
 	consulutil.InitConsulPublisher(maxPubSub, kv)
-	return nil
+
+	// Load stores in order to allow plugins to access different stores
+	return storage.LoadStores(cfg)
```

Comment:
How this will work?

Should delegate/operation plugins themselves be compiled in plugin mode to have access to stores plugins?

Reply:
Yes, you need to compile delegate/operations plugins with CGO_ENABLED=1